doc/usage/configuration.rst

```diff
@@ -354,14 +357,19 @@ General configuration
    * ``ref.python``
    * ``misc.highlighting_failure``
    * ``toc.circular``
+   * ``toc.duplicated``
    * ``toc.excluded``
+   * ``toc.glob``
+   * ``toc.no_title``
+   * ``toc.not_included``
    * ``toc.not_readable``
    * ``toc.secnum``
 
    You can choose from these types.  You can also give only the first
    component to exclude all warnings attached to it.
 
-   Now, this option should be considered *experimental*.
+   It's recommended to use ``ext_*`` type for community extensions.
```

Comment:
I would say that it's perhaps even better to actually use `ext/NAME` for instance. I am not a fan of something like `ext_my.ext` and would prefer seeing something like `ext/my.ext` or `contrib/my.ext`. 

What do other people think?

Reply:
Personally I like underline and slash both, but there is **only** slash in the existing subtype.